### ISSUE TITLE
`v2.0.0-beta.3` panic: toml: local datetimes are expected to have the format YYYY-MM-DDTHH:MM:SS[.NNNNNNNNN]

### ISSUE THREAD

**Issue by valord577** [pelletier/go-toml#589]
**Describe the bug**
...

**To Reproduce**
```go
package main

import (
    "testing"

    "github.com/pelletier/go-toml/v2"
)

var m map[string]interface{}

func TestTomlIssue1(t *testing.T) {
    err := toml.Unmarshal([]byte("dob1 = 07:32:00"), &m)
    if err != nil {
        panic(err)
    }
}
```

<img width="695" alt="Screen Shot 2021-09-08 at 20 01 03" src="https://user-images.githubusercontent.com/30427460/132506607-31eefa0a-f16a-406b-ab80-8e5b0fc704ae.png">

**Expected behavior**
Please check doc: https://toml.io/en/v1.0.0#local-time

**Versions**
 - go-toml: v2.0.0-beta.3
 - go: go version go1.16.6 linux/amd64
 - operating system: Fedora release 34 (Thirty Four)

**Additional context**
...

**Comment by pelletier** [comment]
Hi! Can you try the latest `v2` sha? I believe this has been fixed by https://github.com/pelletier/go-toml/pull/567.

**Comment by moorereason** [comment]
I can confirm that this is fixed in a0d685d4822f.  See https://play.golang.org/p/MoW8zlwN9LM

**Comment by valord577** [comment]
Ho! Thanks! This is a duplicate issue.
